README.md

```diff
@@ -2,6 +2,8 @@
 
 ## About
 
+**WARNING: The goal of this package is to bring together all the JavaScript-related packages developed for interBTC, and wrap around calls that need extra processing for strong typing. Because this is still a work in progress, `@interlay/interbtc-index-client` still needs to be imported explicitly.**
```

Comment:
To be fair it doesn't necessarily always need to be imported explicitly, only if you need specific type definitions that aren't re-exported yet.

Reply:
Well, better to have it and not need it then